setup.py

```diff
@@ -1,21 +1,4 @@
-"""
-    Setup file for Cortex CLI.
-    Use setup.cfg to configure your project.
-
-    This file was generated with PyScaffold 4.0.1.
-    PyScaffold helps you to put up the scaffold of your new Python project.
-    Learn more under: https://pyscaffold.org/
-"""
 from setuptools import setup
 
 if __name__ == "__main__":
-    try:
-        setup(use_scm_version={"version_scheme": "no-guess-dev"})
-    except:  # noqa
-        print(
-            "\n\nAn error occurred while building the project, "
-            "please ensure you have the most updated version of setuptools, "
-            "setuptools_scm and wheel with:\n"
-            "   pip install -U setuptools setuptools_scm wheel\n\n"
-        )
-        raise
+    setup()
```

Comment:
Can't we just remove `setup.py` entirely?

Reply:
I was not sure... I checked EXA repositories and consulted Ozzy, they have dummy ```setup.py``` everywhere:

"Unfortunately, the lowest pip and python versions we support require a dummy setup.py for pip install to work. We'll re-evaluate after finishing the 3.11 migration, but we'll probably have to keep them around for another minor version or two, until we mostly drop 3.9 support"

Any idea how can I test this before merging?

Reply:
I have removed ```setup.py``` after testing build and installation with latest ```pip``` versions